src/configureWorkspace/configure.ts

```diff
@@ -57,8 +57,14 @@ export function getExposeStatements(ports: number[]): string {
     return ports ? ports.map(port => `EXPOSE ${port}`).join('\n') : '';
 }
 
-export function getComposePorts(ports: number[]): string {
-    return ports && ports.length > 0 ? '    ports:\n' + ports.map(port => `      - ${port}:${port}`).join('\n') : '';
+export function getComposePorts(ports: number[], debugPort?: number): string {
+    let portMappings: string[] = ports?.map(port => `      - ${port}`);
```

Comment:
Since `?.` would return `undefined` if `ports` is `undefined`, we should end this line with `?? []`, so that we can be certain `portMappings` will be defined (even if empty).

Reply:
At some point we should start adding the strict-null-check TS compiler flag; this might help flag such situations.